src/common/queries/common/params.interface.ts

```diff
@@ -14,4 +14,5 @@ export interface Params {
   filter?: string;
   status?: string[];
   sort?: string;
+  companyDocuments?: string;
```

Comment:
This can be improved. You can pass `"true" | "false"` or `boolean`, but don't forget to stringify it later.